src/main/java/com/altimetrik/sentimentanalysis/controllers/UserController.java

```diff
@@ -0,0 +1,61 @@
+package com.altimetrik.sentimentanalysis.controllers;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.altimetrik.sentimentanalysis.models.User;
+import com.altimetrik.sentimentanalysis.repositories.UserRepository;
+
+@RestController
+@RequestMapping("/users/")
+public class UserController {
+	
+	@Autowired
```

Comment:
Lets use service classes, for all DB operations, instead of using repository classes in controllers